### DISCUSSION TITLE
How can I customize the email users receive when signing up with magic link?

### DISCUSSION THREAD

**Discussion by lumenwrites** [supabase/supabase#7675] [Questions] [answered]
Is it possible to change the text of the email the users receive when they sign up with magic link?

**Comment by Hallidayo** [comment, marked as answer]
Hi @lumenwrites

On your dashboard for your project if you go to the following:

Authentication > Templates

If you then scroll down you will see a template for Magic Link email sign ups where you can change both the body and subject of the email.

**Reply by lumenwrites** [reply]
Thank you very much for your help, this worked!

Do you know if it's possible to change the sender and sender's name?

Right now the emails are sent from "noreply" from "noreply@mail.app.supabase.io". I'd like to send them from "My App Name" from "support@mywebsite.io".

**Reply by Hallidayo** [reply]
You'd have to provide your own SMTP server details, To do this you go to the same area (Authentication > Templates) and their is a section at the very top of the page where you can figure your SMTP server:

<img width="875" alt="Screenshot 2022-07-14 at 20 54 44" src="https://user-images.githubusercontent.com/22655069/179072003-b96c6bd3-e4f0-4cd8-bace-d7972bd0dbaf.png">

**Reply by lumenwrites** [reply]
Thank you so much!
